### ISSUE TITLE
Incorrect rendering of DWORD

### ISSUE THREAD

**Issue by msiecx** [ajkhoury/ReClassEx#50]
There is a problem rendering DWORD values in your build of reclass:
https://i.imgur.com/f4O95Gp.png

In CNodeDWORD.cpp:

```
//tx = AddText( View, tx, y, g_clrValue, HS_EDIT, g_bUnsignedHex ? _T( "0x%IX" ) : _T( "%u" ), *Data ) + g_FontWidth;
tx = AddText( View, tx, y, g_clrValue, HS_EDIT, g_bUnsignedHex ? _T( "0x%lX" ) : _T( "%u" ), *Data ) + g_FontWidth;
```

IX (uppercase I) is for ULONG_PTR, you want lX (lower case L) for unsigned int32.

**Comment by ajkhoury** [comment]
Nice catch! If you could open a PR, that would be 💯

**Comment by ajkhoury** [comment]
Fixed in 3cdc590
